docs/spark-standalone.md

```diff
@@ -93,6 +93,10 @@ You can optionally configure the cluster further by setting environment variable
 <table class="table">
   <tr><th style="width:21%">Environment Variable</th><th>Meaning</th></tr>
   <tr>
+    <td><code>SPARK_MASTER</code></td>
```

Comment:
Thanks for giving suggestion. Configuration file  synchronization  is a very common requirement, why not recommend? Do we need add some comment to explain it ?